### PR TITLE
Fix flacky test

### PR DESCRIPTION
**Pull request by tivv**

### CODE REVIEW

File: cdap-unit-test/src/main/java/io/cdap/cdap/test/TestBase.java [hunk 1]
```diff
@@ -105,7 +104,6 @@
 import io.cdap.cdap.metadata.MetadataService;
 import io.cdap.cdap.metadata.MetadataServiceModule;
 import io.cdap.cdap.metadata.MetadataSubscriberService;
-import io.cdap.cdap.metadata.PreferencesFetcher;
```
Comment (greeshmaswaminathan) [inline cdap-unit-test/src/main/java/io/cdap/cdap/test/TestBase.java:108]: Unused imports ?

**Review by greeshmaswaminathan — APPROVED** [review]

File: cdap-unit-test/src/main/java/io/cdap/cdap/test/TestBase.java (hunk 1 above)
Comment (tivv) [reply cdap-unit-test/src/main/java/io/cdap/cdap/test/TestBase.java:108]: Probably. I have autooptimize enabled.

**Comment by greeshmaswaminathan** [comment]
Looks like the SYSTEM_PROGRAM_SCAN_INTERVAL_SECONDS value has to be set to original value for DataPipelineServiceTest.

**Comment by tivv** [comment]
Well, stopping service should fix it. I don't want to decrease timeout as it would mean waiting for it and tests should not wait, but should finish fast

**Pull request merged**